addons/mass_mailing_event/models/event_event.py

```diff
@@ -16,7 +16,9 @@ def action_mass_mailing_attendees(self):
             'target': 'current',
             'context': {
                 'default_mailing_model_id': self.env.ref('event.model_event_registration').id,
-                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', 'not in', ['cancel', 'draft'])])
+                'default_mailing_domain': repr([('event_id', 'in', self.ids), ('state', 'not in', ['cancel', 'draft'])]),
+                'mailing_model_id': self.env.ref('event.model_event_registration').id,
+                'mailing_domain': repr([('event_id', 'in', self.ids), ('state', 'not in', ['cancel', 'draft'])]),
```

Comment:
do we really need that many context key ?
can `default_mailing_domain` be removed if we rely on `_mailing_get_default_domain` ?